.github/SECURITY.md

```diff
@@ -0,0 +1,12 @@
+# Reporting security issues
+
+The project maintainers take security seriously. If you discover a security
+issue, please bring it to their attention right away!
+
+**Please _DO NOT_ file a public issue**, instead send your report privately to
+[security@docker.com](mailto:security@docker.com).
+
+Security reports are greatly appreciated, and we will publicly thank you for it.
```

Comment:
Not a blocker (we need to update in moby as well), but we should mention that we publicly thank them _if they want to_ (some people may not want to, or their employer may not allow them to be mentioned)